tutorindigo/plugin.py

```diff
@@ -107,8 +108,10 @@ def _override_openedx_docker_image(
             "mfe-dockerfile-post-npm-install-learning",
             """
 RUN npm install '@edx/brand@npm:@edly-io/indigo-brand-openedx@^2.0.0'{% if INDIGO_ENABLE_DARK_THEME %} --theme=dark{% endif %}
-RUN npm install '@edx/frontend-component-header@npm:@edly-io/indigo-frontend-component-header@^3.0.0'
-RUN npm install '@edx/frontend-component-footer@npm:@edly-io/indigo-frontend-component-footer@^2.0.0'
+RUN npm install '@edx/frontend-component-header@npm:@edly-io/indigo-frontend-component-header@~3.0.0'
```

Comment:
Can you add a Python comment to explain what is going on here?

Reply:
Sure. We're restricting MFEs to install header from version `3.0.0` to `3.0.x`